docs/docsite/rst/roadmap/COLLECTIONS_13.rst

```diff
@@ -0,0 +1,81 @@
+..
+   THIS DOCUMENT IS OWNED BY THE ANSIBLE COMMUNITY STEERING COMMITTEE. ALL CHANGES MUST BE APPROVED BY THE STEERING COMMITTEE!
+   For small changes (fixing typos, language errors, etc.) create a PR and ping @ansible/steering-committee.
+   For other changes, create a discussion as described in https://ansible.readthedocs.io/projects/ansible/devel/community/steering/community_topics_workflow.html#creating-a-topic
+   to discuss the changes.
+   (Creating a draft PR for this file and mentioning it in the community topic is also OK.)
+
+.. _ansible_13_roadmap:
+
+====================
+Ansible project 13.0
+====================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-core <https://pypi.org/project/ansible-core/>`_ package as well. All dates are subject to change. See the `ansible-core 2.20 Roadmap <https://ansible.readthedocs.io/projects/ansible/devel/roadmap/ROADMAP_2_20.html>`_ for the most recent updates on ``ansible-core``.
+
+.. contents::
+   :local:
+
+
+Release schedule
+=================
+
+The schedule has been updated to match the :ref:`ansible-core 2.20 release schedule <core_roadmap_2.20>`. If that one gets modified again, the Ansible 12 pre-release and release dates will be shifted accordingly, and potentially more pre-releases will be inserted.
+
+:2025-09-15: ansible-core feature freeze.
+:2025-09-22: Start of ansible-core 2.20 betas.
+:2025-09-17: Ansible-13.0.0 alpha1; there will be more alpha releases usually following additional ansible-core 2.20 betas.
```

Comment:
I'm not sure what I was thinking, but it should be the 23rd, not the 17th, especially since the 17th is a Wednesday and *before* the first ansible-core 2.20 beta... :facepalm:
```suggestion
:2025-09-23: Ansible-13.0.0 alpha1; there will be more alpha releases usually following additional ansible-core 2.20 betas.
```